packages/components/src/components/CreateCard/CreateCard.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { Icon, IconNames } from '../Icon';
+import { Stack } from '../Stack';
+import { Text } from '../Text';
+
+type StackProps = React.ComponentProps<typeof Stack>;
+
+type CreateCardProps = {
+  icon: IconNames;
+  label: string;
+} & StackProps;
+
+export const CreateCard: React.FC<CreateCardProps> = ({
+  as = 'button',
+  icon,
+  label,
+  ...props
+}) => {
+  return (
+    <Stack
+      as={as}
+      css={{
+        position: 'relative',
+        overflow: 'hidden',
+        maxWidth: '276px',
```

Comment:
Preferably the width / height is just constrained by the layout and context around the component instead of the component itself. Since this will be usually used inside a grid I think we can get rid of the `maxWidth` and make the component more flexible.

Reply:
@tristandubbeld I thought about tweaking this when it gets implemented inside the grid, but I removed it already here: 1415aca3e70a04e7508ae8916795f638c83c7b3e